test/state/state.cpp

```diff
@@ -415,48 +420,32 @@ StateDiff finalize(const StateView& state_view, evmc_revision rev, const address
     return state.build_diff(rev);
 }
 
-std::variant<TransactionReceipt, std::error_code> transition(const StateView& state_view,
-    const BlockInfo& block, const BlockHashes& block_hashes, const Transaction& tx,
-    evmc_revision rev, evmc::VM& vm, int64_t block_gas_left, int64_t blob_gas_left)
+TransactionReceipt transition(const StateView& state_view, const BlockInfo& block,
+    const BlockHashes& block_hashes, const Transaction& tx, evmc_revision rev, evmc::VM& vm,
```

Comment:
I would change `tx` to `validated_tx`

Reply:
That's another ~20 changes. I think not, if anything we can try to encode this with a type.

Reply:
OK.